__tests__/app.test.js

```diff
@@ -26,8 +27,24 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api", () => {
+  it("status 200, should return an object describing all the available endpoints on your API", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        const { endpoints } = body;
+        fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((file) => {
```

Comment:
Although this works you have a nested .then which we try to avoid. There is no reason you can't just require in the file to the test suite and assert it is sent back by the endpoint in the test - without having to use fs in the test.